### ISSUE TITLE
Scrubber should skip templates

### ISSUE THREAD

**Issue by khenderick** [openvstorage/framework#985]
Scrubber should skip templates

```
Traceback (most recent call last):
  File "/opt/OpenvStorage/ovs/lib/scheduledtask.py", line 309, in _execute_scrub_work
    work_units = locked_client.get_scrubbing_workunits()
  File "/usr/lib/python2.7/dist-packages/rpyc/core/netref.py", line 196, in __call__
    return syncreq(_self, consts.HANDLE_CALL, args, kwargs)
  File "/usr/lib/python2.7/dist-packages/rpyc/core/netref.py", line 71, in syncreq
    return conn.sync_request(handler, oid, *args)
  File "/usr/lib/python2.7/dist-packages/rpyc/core/protocol.py", line 441, in sync_request
    raise obj
InvalidOperationException: Throw location unknown (consider using BOOST_THROW_EXCEPTION)
Dynamic exception type: volumedriverfs::InvalidOperationException
std::exception::what: std::exception
[volumedriverfs::tag_volume_id*] = 187dd671-0236-47dc-895a-ab5e3168a9f1
[volumedriverfs::tag_desc*] = invalid object type
```

**Comment by khenderick** [comment]
Fixed by #997, packaged by openvstorage-2.7.4-rev.4086.a89c9cc

**Comment by kinvaris** [comment]
PASSED:
- Created a vdisk
- Wrote some data on it
- Made it a vTemplate
- Cloned 1 disk
- Started scrubbing
- Result
